frontend/src/main/web/lib/stores/UserMatrixStore.js

```diff
@@ -4,7 +4,7 @@ import {Promise} from 'es6-promise';
 import {EventEmitter} from 'events';
 import {ContentStates} from '../constants/Options';
 import {DateRanges} from '../constants/Options';
```

Comment:
Combine these to `import { ContentStates, DateRanges } from '../constants/Options'`